trade-finance-manager-ui/component-tests/assertions.ts

```diff
@@ -107,7 +107,7 @@ export const assertions = <TParams extends object>(wrapper: CheerioAPI, html: st
     },
   }),
   expectInput: (selector: string) => ({
-    toHaveValue: (value: string) => {
+    toHaveValue: (value?: string) => {
```

Comment:
Can the param remain required?

Reply:
This is from the JS->TS migration, tried leaving this as is initially, but the input value the component tests return when no value is given is `undefined`.

We could have an additional function on `expectInput`, e.g. `toHaveAnUndefinedValue`, but I'm not super keen on that. 

Some debug after changing the expected values to empty string to match the typing (instead of undefined):
![Screenshot 2024-11-05 at 13 54 50](https://github.com/user-attachments/assets/89e157c0-9797-439e-ba45-66d02524119e)
![Screenshot 2024-11-05 at 13 54 55](https://github.com/user-attachments/assets/d62b8d4a-7f7f-4222-8193-f60e90ee99fe)